bears/scss/SCSSLintBear.py

```diff
@@ -1,5 +1,6 @@
 import yaml
 
+from coalib.bearlib import deprecate_settings
```

Comment:
Why did you import this file?

Reply:
@shreyateeza because I needed to deprecate a setting. Read the issue.